### PR TITLE
IGAPP-452: Apply prettier rules

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

The pre-commit hook is available, but for only if you clone the project. It's not possible to share it for someone who already cloned the repo. If you want to use it, just copy it from .github/hooks/pre-commit to .git/hooks/pre-commit. You can modify it, to also check linting, flow or test, before commit, as you prefer.

For reviewing have a look at these two commits:
[IGAPP-452: Add pre-commit-hook and prettier checks to pipeline](https://github.com/Integreat/integreat-app/pull/197/commits/19b464c0796a3f037121f10dc1a9d64649653f19)
[IGAPP-452: Apply prettier rules on project level](https://github.com/Integreat/integreat-app/pull/197/commits/b34b3491451164f4404e26126713fbb9cfcb6069)

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.circleci/
+build-configs/
```
Comment (klinzo) [inline .prettierignore:2]: I think we can use prettier for build configs, docs, release notes, tools and translations as well. Or did you have a reason to exclude it?

File: native/android/fastlane/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-fastlane documentation
-================
+# fastlane documentation
```
Comment (klinzo) [inline native/android/fastlane/README.md:1]: I think these fastlane readmes are autogenerated and should maybe be excluded.

File: native/metro.config.ci.js [hunk 3]
```diff
@@ -2,9 +2,11 @@ const defaultConfig = require('./metro.config.js')
...
 module.exports = {
   ...defaultConfig,
-  maxWorkers: process.env.TOTAL_CPUS || 1, /* CircleCI: The issue with this on Dockerized services such as CircleCI is
+  maxWorkers:
+    process.env.TOTAL_CPUS ||
+    1 /* CircleCI: The issue with this on Dockerized services such as CircleCI is
                                               that the container thinks it has access to all of the hosts cores,
                                               although its being limited down to 4 by another process.
-                                              https://github.com/facebook/create-react-app/issues/4870 */
+                                              https://github.com/facebook/create-react-app/issues/4870 */,
```
Comment (klinzo) [inline native/metro.config.ci.js:10]: This is weird but probably just fine

**Review by klinzo — APPROVED** [review]
Linting for md files, I like it :partying_face: 
I think I have to get used to no space after function names but apart from that I am a fan!

Only visited files at random.

File: .prettierignore (hunk 1 above)
Comment (SteffiStuffel) [reply .prettierignore:2]: We have no valid point to check them in the pipeline, we do only check web and check native. Therefore i did not add. If i just overlook it, we can add it. But lets add it to a new issue. I would love to have this merged asap.

File: web/src/modules/common/components/Tooltip.js [hunk 4]
```diff
@@ -38,64 +38,79 @@ type FlowType = 'left' | 'right' | 'up' | 'down'
...
     `}
   }
-  
+
   ::after {
-    ${flow === 'up' && `
+    ${flow === 'up' &&
+    `
       bottom: calc(99% + 5px);
     `}
```
Comment (Taggotty) [inline web/src/modules/common/components/Tooltip.js:71]: those expressions actually seem uneccessary lengthy. But didn't find an option to not break quotes, so I think it is fine.

**Review by Taggotty — APPROVED** [review]
Some parts become longer, some shorter, but that's fine to me.

File: native/android/fastlane/README.md (hunk 2 above)
Comment (SteffiStuffel) [reply native/android/fastlane/README.md:1]: done

Comment (SteffiStuffel) [reply native/android/fastlane/README.md:1]: done

**Pull request merged**